samples/README.md

```diff
@@ -40,13 +40,21 @@ terraform destroy
 popd
 ```
 
-## Kubectl
 
-- TODO
+# Kubectl
 
-## Helm
+```bash
+# Check Kubectl version
+kubectl version
+```
 
-- TODO
+# Helm
```

Comment:
Can you keep the heading of Helm and kubectl as is? Just to ensure the right format as other tools. Thanks
```suggestion
## Helm
```